File: Sources/Assets/Scripts/TextControl.js
#pragma strict
var isQuitButton = false;
var isPlayButton = false;
var isInstructionButton = false;

function OnMouseEnter(){
	renderer.material.color = Color.green;
}

function OnMouseExit(){
	renderer.material.color = Color.white;
}

function OnMouseUp(){
	if(isQuitButton){
		Application.Quit();
	}
	if(isPlayButton){
		Application.LoadLevel(0);
	}
	if(isInstructionButton){
		Application.LoadLevel(2);
	}
}

File: Sources/Assets/Scripts/TextControl.cs
using UnityEngine;
using System.Collections;

public class TextControl : MonoBehaviour {
public bool isQuitButton = false;
public bool isPlayButton = false;
public bool isInstructionButton = false;

	void OnMouseEnter(){
		gameObject.renderer.material.color = Color.green;
	}

void OnMouseExit(){
		gameObject.renderer.material.color = Color.white;
}

void OnMouseUp(){
	if(isQuitButton){
		Application.Quit();
	}
	if(isPlayButton){
			timer.time = 300;
			Health_Player.vie_courant = 5;
			LanceurBatteReondissanteCSharp.nbMaxMunition = 10;
			Application.LoadLevel(0);
	}
	if(isInstructionButton){
			Application.LoadLevel(2);
	}
}
}

